src/richie/apps/core/api.py

```diff
@@ -0,0 +1,45 @@
+"""
+API endpoints for the core app.
+"""
+from rest_framework.decorators import action
+from rest_framework.response import Response
+from rest_framework.viewsets import ViewSet
+
+from .serializers import UserSerializer
+
+
+class UsersViewSet(ViewSet):
+    """
+    API endpoints to access users.
+    """
+
+    # pylint: disable=no-self-use,unused-argument
+    def list(self, request, version):
```

Comment:
I think you should return 405 http codes for methods that are not supported. Isn't it what DRF returns if you just don't declare any list method on the class?

Reply:
IMO `405` is the wrong code to return here. It's not as if we support `POST` or `PUT` or whatever on the endpoints.

If `501` is not what we want then we're probably better off with `404` as @lunika suggests?

Reply:
DRF returns a `404` if the methods are not implemented. I'll just do the same here.

Reply:
so just remove these methods from you class?